packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -1080,55 +1076,27 @@ describe(componentName, () => {
 
   //Empty State
   it('renders an empty table', async () => {
-    render(<EmptyUsage data-testid={dataTestId}></EmptyUsage>);
+    const { rerender } = render(<EmptyUsage data-testid={dataTestId} />);
+    screen.getByText('Empty State Title');
+    screen.getByText('Description test explaining why this card is empty.');
     expect(
-      screen.getByRole('table').getElementsByTagName('tbody')[0].className
-    ).toEqual('c4p--datagrid__empty-state-body');
+      document.querySelector(`.${pkg.prefix}--empty-state-type--noData`)
```

Comment:
We should avoid `document.querySelector` if possible

Reply:
i'll take another look